ee/ucp/admin/configure/collect-cluster-metrics.md

```diff
@@ -84,8 +84,12 @@ To deploy Prometheus on worker nodes in a cluster:
     $ kubectl label node 3a724a-1 ucp-metrics=
     node "test-3a724a-1" labeled
     ```
+    
+     > **SELinux Prometheus Deployment**
```

Comment:
@wsong should we add UCP versions this note  is relevant to given [this comment](https://github.com/docker/orca/issues/15590#issuecomment-446024780)?

Reply:
Sure, that's a good idea.